.changeset/smooth-ads-admire.md

```diff
@@ -0,0 +1,6 @@
+---
+"@twilio-paste/side-modal": major
+"@twilio-paste/core": major
```

Comment:
Check with @nkrantz but if it's not a code breaking change and style should this just be a minor?

```suggestion
"@twilio-paste/side-modal": minor
"@twilio-paste/core": minor
```

Reply:
yes minor

Reply:
also might be nice to have a bit more context in the changeset as to why we're making this change and what design action is needed

Reply:
@nkrantz changeset updated